README.md

```diff
@@ -51,7 +51,7 @@ If you think you want to contribute to the bot, make sure to visit the
 
 ## License
 ```
-   Copyright 2017 Kaidan Gustave
+   Copyright 2017-2018 Kaidan Gustave
```

Comment:
I would like you to revert this copyright change.
This is an action reserved only for the licenseholder, which you are not.